src/assets/i18n/en_US.json

```diff
@@ -934,6 +991,18 @@
   "navigation.paging.go_to_page.label": "Go to page {{0}}",
   "navigation.paging.next_page.label": "Go to next page",
   "navigation.paging.previous_page.label": "Go to previous page",
+  "number.decrease.text": "-",
+  "number.increase.text": "+",
+  "order.recurrence.end.label": "End",
```

Comment:
@M-Behr Shouldn't this be "Until after" according to Figma? Actually, I would suggest "End after" to say "End after 50 orders".